testing/fuzzing/bootstrap_harness.cc

```diff
@@ -3,19 +3,131 @@
+        dispatch, [](Tox *tox, const Tox_Event_File_Recv_Chunk *event, void *user_data) {
+            assert(event == nullptr);
+        });
+    tox_events_callback_file_recv_control(
+        dispatch, [](Tox *tox, const Tox_Event_File_Recv_Control *event, void *user_data) {
+            assert(event == nullptr);
+        });
+    tox_events_callback_friend_connection_status(
+        dispatch, [](Tox *tox, const Tox_Event_Friend_Connection_Status *event, void *user_data) {
+            assert(event == nullptr);
+        });
+    tox_events_callback_friend_lossless_packet(
+        dispatch, [](Tox *tox, const Tox_Event_Friend_Lossless_Packet *event, void *user_data) {
+            assert(event == nullptr);
+        });
+    tox_events_callback_friend_lossy_packet(
+        dispatch, [](Tox *tox, const Tox_Event_Friend_Lossy_Packet *event, void *user_data) {
+            assert(event == nullptr);
+        });
+    tox_events_callback_friend_message(
+        dispatch, [](Tox *tox, const Tox_Event_Friend_Message *event, void *user_data) {
+            assert(event == nullptr);
+        });
+    tox_events_callback_friend_name(
+        dispatch, [](Tox *tox, const Tox_Event_Friend_Name *event, void *user_data) {
+            assert(event == nullptr);
+        });
+    tox_events_callback_friend_read_receipt(
+        dispatch, [](Tox *tox, const Tox_Event_Friend_Read_Receipt *event, void *user_data) {
+            assert(event == nullptr);
+        });
+    tox_events_callback_friend_request(
+        dispatch, [](Tox *tox, const Tox_Event_Friend_Request *event, void *user_data) {
+            Tox_Err_Friend_Add err;
+            tox_friend_add_norequest(tox, tox_event_friend_request_get_public_key(event), &err);
+            assert(err == TOX_ERR_FRIEND_ADD_OK || err == TOX_ERR_FRIEND_ADD_OWN_KEY
+                || err == TOX_ERR_FRIEND_ADD_ALREADY_SENT
+                || err == TOX_ERR_FRIEND_ADD_BAD_CHECKSUM);
+        });
+    tox_events_callback_friend_status(
+        dispatch, [](Tox *tox, const Tox_Event_Friend_Status *event, void *user_data) {
+            assert(event == nullptr);
+        });
+    tox_events_callback_friend_status_message(
+        dispatch, [](Tox *tox, const Tox_Event_Friend_Status_Message *event, void *user_data) {
+            assert(event == nullptr);
+        });
+    tox_events_callback_friend_typing(
+        dispatch, [](Tox *tox, const Tox_Event_Friend_Typing *event, void *user_data) {
+            assert(event == nullptr);
+        });
+    tox_events_callback_self_connection_status(
+        dispatch, [](Tox *tox, const Tox_Event_Self_Connection_Status *event, void *user_data) {
+            assert(event == nullptr);
+        });
+}
+
+}
+
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size);
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 {
+    if (size < 1) {
+        return 0;
+    }
+    const uint8_t iterations = data[0];
```

Comment:
With this you basically invalidate the existing fuzzer corpus. I like your idea to read until the end of the fuzzer stream.

Reply:
Done that now.